src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/MetadataUtilsSmi.cs

```diff
@@ -439,7 +500,8 @@ internal static SqlMetaData SmiExtendedMetaDataToSqlMetaData(SmiExtendedMetaData
                     source.TypeSpecificNamePart2,
                     source.TypeSpecificNamePart3,
                     true,
-                    source.Type);
+                    source.Type
```

Comment:
Could you check if "SmiExtendedMetaDataToSqlMetaData" function hasn't been used and could be removed?

Reply:
It was not being used. I removed it.